tools/release_generator/release.py

```diff
@@ -71,7 +68,7 @@
    Framework <framework>
    User Interface & Usability <ui>
    Diffraction <diffraction>
-   LowQ <lowq>
+   SANS <sans>
```

Comment:
So what happens to reflectometry?

Reply:
Good point. I was just following our current layout. It looks like we actually have separate pages for SANS & Reflectometry currently so I think I'll go with that.